cassandra/policies.py

```diff
@@ -376,7 +380,36 @@ def make_query_plan(self, working_keyspace=None, query=None):
                 for host in child.make_query_plan(keyspace, query):
                     yield host
             else:
-                replicas = self._cluster_metadata.get_replicas(keyspace, routing_key)
+                if EXPERIMENTAL_TABLETS and self._cluster_metadata.use_tablets():
```

Comment:
Yeah, you are right, I removed this check, but there is one doubt, because if user does not enable explicitly that he want to use experimental tablets I do not think we should enter this parts of code, so I do not know if it is correct to call use_tablets without additional check. It originates from the idea that cluster can have tablets enabled or not (we can say that `use_tablets` checks that) and driver should use tablets only if user enable it (whether or not it is enabled at the cluster level, my idea was to have EXPERIMENTAL_TABLETS env for this check)

Reply:
After other changes we discussed, you should be able to change this condition to just checking if tablets were enabled in the `Cluster` object.

Reply:
Done